artemis-console-extension/artemis-extension/packages/artemis-console-plugin/src/queues/QueuesView.tsx

```diff
@@ -99,6 +101,62 @@ export const QueuesView: React.FunctionComponent<Navigate> = navigate => {
         <MessageView currentMessage={currentMessage} back={back}/>
         </>
         }
+        <Modal aria-label='copy-message-modal'
+                variant={ModalVariant.medium}
+                isOpen={showHelpModal}
+                actions={[
+                  <Button key="cancel" variant="secondary" onClick={() => setShowHelpModal(false)}>
+                    Close
+                  </Button>
+                ]}>
+          <TextContent>
+            <Text component={TextVariants.h1}>Using Filter Expressions</Text>
+            <Text component={TextVariants.h2}>Apache ActiveMQ Artemis provides a powerful filter language based on a subset of the SQL 92 expression syntax.</Text>
+            <Text>It is the same as the syntax used for JMS & Jakarta Messaging selectors. 
+              For documentation on JMS selector syntax please the JavaDoc for javax.jms.Message or jakarta.jms.Message</Text>
+              <Text>A Filter will search for messages that have a matching header.</Text>
+              <Text component={TextVariants.h3}>Filter Examples</Text>
+              <TextList component={TextListVariants.dl}>
+                <TextListItem component={TextListItemVariants.dt}>header = 'value'</TextListItem>
+                <TextListItem component={TextListItemVariants.dd}> This will return any message where the message header <b>header</b> has the string value <b>value</b></TextListItem>
```

Comment:
Saying the syntax is the same as (possibly say 'similar to', as it is not actually exactly the same?) JMS/Jakarta Messaging selectors and then immediately saying filters matches 'headers' possibly has potential to confuse given matching will as/more often be done against JMS _properties_; only a handful of JMS headers are allowed to be used in selectors, most of which arent mentioned here.